apps/website/src/pages/docs/carousel.mdx

```diff
@@ -61,13 +61,13 @@ This component uses a composition approach, similar to most of the other compone
 
 ### Navigation
 
-The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right, and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
+The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right and the `Carousel.DotsList` component shown in the middle. The navigation section must be present before the slides in DOM order.`children` can be specified to override what is shown in this navigation section.
```

Comment:
Can the **"must"** part somehow be given more emphasis? Maybe we create an alert type component for this important piece of information? E.g. from the Astro Docs ([link](https://docs.astro.build/en/guides/styling/#import-a-local-stylesheet)):

![image](https://github.com/iTwin/iTwinUI/assets/45748283/9589029c-1dfd-4148-8708-36053eed120e)

To do this, maybe we can re-purpose the `<Placeholder>`?

![image](https://github.com/iTwin/iTwinUI/assets/45748283/950f5e28-bd7e-4758-954c-debd5d6f9c75)

But for this PR at least, in the interest of time, you can just make the "must" portion emphasized some way. Maybe bold text? The Alert type can come later. Maybe we can make a mention of this idea in #961 to remember for the future.

Reply:
New paragraph starting with a bold "**Important:**"?